test/Microsoft.VisualStudio.Composition.Analyzers.Tests/VSMEF006ImportNullabilityAnalyzerTests.cs

```diff
@@ -0,0 +1,421 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.VisualStudio.Composition.Analyzers;
+using VerifyCS = CSharpCodeFixVerifier<Microsoft.VisualStudio.Composition.Analyzers.VSMEF006ImportNullabilityAnalyzer, Microsoft.VisualStudio.Composition.Analyzers.CodeFixes.VSMEF006ImportNullabilityCodeFixProvider>;
+
+public class VSMEF006ImportNullabilityAnalyzerTests
```

Comment:
All of these tests are based on importing reference types. We should test importing of value types and assert that none of them are ever expected to bear the `?` type suffix regardless of `AllowDefault` values.

Reply:
Thanks! Fixed. `int?` and `int` are now correctly treated as separate types, and the presence or absence of `AllowDefault` has no impact on the analyzer.